graphql_compiler/typedefs.py

```diff
@@ -4,6 +4,17 @@
 from graphql import GraphQLList, GraphQLNonNull, GraphQLScalarType
 
 
+# The below code is an import shim for TypedDict: we don't want to conditionally import it from
+# every file that needs it. Instead, we conditionally import it here and then import from this file
+# in every other location where this is needed.
+#
+# Hence, the "unused import" warnings on TypedDict here are false-positives.
+try:
+    from typing import TypedDict  # noqa  # pylint: disable=unused-import
+except ImportError:  # TypedDict was only added to typing in Python 3.8
+    from typing_extensions import TypedDict  # noqa  # pylint: disable=unused-import
```

Comment:
Looks like this is a `no fix` sort of situation btw: https://github.com/python/mypy/issues/1107

Reply:
Thankfully, I think `mypy` doesn't mind in this case. Another positive in this situation is that `TypedDict` has the same signature no matter where it is imported from.